src/entity/mesh.hpp

```diff
@@ -50,7 +49,7 @@ class Mesh {
         indexed_vertices_(indexed_vertices), indexed_color_ids_(indexed_color_ids),
         indexed_normals_(indexed_normals), color_map_(color_map) {}
 
-    friend World;
+    //friend World;
```

Comment:
delete

Reply:
The goal was to set `change_color_indexing` back to private when `world::get_mesh_greedy` is removed in the next pr. This comment is lower in the file though. In this case we include the world header, and set `change_color_indexing` to private. Now `World` can only change the color indexing if it is declared friend.